app/controllers/api/items_controller.rb

```diff
@@ -11,8 +11,18 @@ def create
     end
   end
 
+  def update
+    @list = List.find(params[:list_id])
```

Comment:
I would extract this out into a helper method so you don't have to make a database call twice.